packages/onboarding/src/utils/flows.ts

```diff
@@ -35,3 +42,19 @@ export const isTailoredSignupFlow = ( flowName: string | null ) => {
 				ECOMMERCE_FLOW === flowName )
 	);
 };
+
+export const isCurrentPlanFlow = ( flowName: string | null ) => {
```

Comment:
As per pdm0RZ-nC-p2#comment-756, the following landing pages have Zendesk chat enabled among others:

* `/do-it-for-me/`
* `/professional-email/`
* `/themes`
* `/domains`

Each of these landing page CTAs links to a dedicated signup flow - `/start/do-it-for-me`, `/start/onboarding-with-email`, `/start/with-theme`, and `/start/domain` - which are not covered in the list below. Just wanted to confirm if this is intentional. I haven't tested but I assume the effect would be that the chat widget would disappear if the user starts from the signup flow linked on those landing pages.